tools/Raven.Migrator/AbstractMigrationConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Raven.Migrator
+{
+    public class AbstractMigrationConfiguration
+    {
+        public string Command { get; set; }
+
+        public string DatabaseName { get; set; }
+
+        public bool ConsoleExport { get; set; }
+
+        public string ExportFilePath { get; set; }
```

Comment:
`ConsoleExport = ExportFilePath == null` ? Can remove that one property?

Reply:
if ExportFilePath == null and ConsoleExport == true a file will be exported in the application folder

Reply:
Looks like a unnecessary side-effect?

Reply:
The tool can work as standalone and export a valid .ravendbdump file.
Raven.Migrator.exe export
{"Command": "export", "DatabaseName": "test" }

If I remove the ConsoleExport property this command will export to the console as ExportFilePath == null

Reply:
Can we export to two of those sources at once?

Reply:
no

Reply:
I think it would be better to throw when both of them are set or none of them is set?